gio/src/file_attribute_value.rs

```diff
@@ -0,0 +1,126 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
```

Comment:
This should be done this way in the first commit, to avoid the noise in this commit.